docs/guides.md

```diff
@@ -1259,7 +1261,11 @@ To remove the installed certs in the pool:
 secureboot-certs clear
 ```
 
-Note that this does not remove the certs from the VMs or from disk. To remove them from disk, remove the ".auth" files for the certs you'd like to remove, on every host (found in `/var/lib/uefistored/`). In order to clear the certs from the VMs it is required to use `varstore-rm`. See [Remove Certificates from a VM](#remove-certificates-from-a-vm).
+:::tip
+Note that this does not remove the certs from the VMs nor from disk on XCP-ng 8.2.
```

Comment:
Ambiguous. One may understand that it's only on XCP-ng 8.2 that certs are not removed from VMs.

I would also write 8.2.x here and below so that people are not confused as to whether the comment applies to 8.2.1 or not.